compiler/qsc_doc_gen/src/generate_docs.rs

```diff
@@ -366,11 +378,19 @@ title: {}
 ms.date: {{TIMESTAMP}}
 ms.topic: {}
 qsharp.kind: {}
+qsharp.package: {}
```

Comment:
This is compatible with documentation consumption for MS learn site, right?

Reply:
(Note: Confirming with the docs team just to be safe. I'll get the Ok before I merge).